packages/sdk/src/types/AccountAddress.ts

```diff
@@ -24,6 +38,8 @@ class AccountAddress {
  * - The base58 string is a length of 50 (encoding exactly 32 bytes).
  */
 export type Type = AccountAddress;
+export const instanceOf = (value: unknown): value is AccountAddress =>
```

Comment:
Documentation would be nice here. Consider defining using `function ...` to follow the style of the rest of the file